--- extras/Makefile.orig	2013-07-30 16:58:35.000000000 +0200
+++ extras/Makefile.in	2013-07-30 17:00:32.000000000 +0200
@@ -66,18 +66,20 @@
 
 
 install:	all
-	rm -f $(BINDIR)/makeweb $(BINDIR)/htpasswd $(BINDIR)/syslogtocern
-	cp makeweb $(BINDIR)/makeweb
-	chgrp $(WEBGROUP) $(BINDIR)/makeweb
-	chmod 2755 $(BINDIR)/makeweb
-	cp htpasswd $(BINDIR)/htpasswd
-	cp syslogtocern $(BINDIR)/syslogtocern
-	rm -f $(MANDIR)/man1/makeweb.1
-	cp makeweb.1 $(MANDIR)/man1/makeweb.1
-	rm -f $(MANDIR)/man1/htpasswd.1
-	cp htpasswd.1 $(MANDIR)/man1/htpasswd.1
-	rm -f $(MANDIR)/man8/syslogtocern.8
-	cp syslogtocern.8 $(MANDIR)/man8/syslogtocern.8
+	rm -f $(DESTDIR)$(BINDIR)/makeweb $(DESTDIR)$(BINDIR)/htpasswd $(DESTDIR)$(BINDIR)/syslogtocern
+	cp makeweb $(DESTDIR)$(BINDIR)/makeweb
+	chgrp $(WEBGROUP) $(DESTDIR)$(BINDIR)/makeweb
+	chmod 2755 $(DESTDIR)$(BINDIR)/makeweb
+	cp htpasswd $(DESTDIR)$(BINDIR)/htpasswd
+	cp syslogtocern $(DESTDIR)$(BINDIR)/syslogtocern
+	mkdir -p $(DESTDIR)$(MANDIR)/man1
+	mkdir -p $(DESTDIR)$(MANDIR)/man8
+	rm -f $(DESTDIR)$(MANDIR)/man1/makeweb.1
+	cp makeweb.1 $(DESTDIR)$(MANDIR)/man1/makeweb.1
+	rm -f $(DESTDIR)$(MANDIR)/man1/htpasswd.1
+	cp htpasswd.1 $(DESTDIR)$(MANDIR)/man1/htpasswd.1
+	rm -f $(DESTDIR)$(MANDIR)/man8/syslogtocern.8
+	cp syslogtocern.8 $(DESTDIR)$(MANDIR)/man8/syslogtocern.8
 
 clean:
 	rm -f $(CLEANFILES)
