--- source/SQLite/sqlite3.c.orig	2013-06-27 16:19:30.000000000 +0200
+++ source/SQLite/sqlite3.c	2013-06-27 16:20:00.000000000 +0200
@@ -22768,13 +22768,6 @@
 */
 #if SQLITE_OS_UNIX              /* This file is used on unix only */
 
-/* Use posix_fallocate() if it is available
-*/
-#if !defined(HAVE_POSIX_FALLOCATE) \
-      && (_XOPEN_SOURCE >= 600 || _POSIX_C_SOURCE >= 200112L)
-# define HAVE_POSIX_FALLOCATE 1
-#endif
-
 /*
 ** There are various methods for file locking used for concurrency
 ** control:
